src/config.rs

```diff
@@ -27,8 +27,13 @@ pub struct Config {
     pub auth: identity::AuthSource,
 
     pub termination_grace_period: time::Duration,
+
+    /// Specify the number of worker threads the Tokio Runtime will use.
+    pub num_worker_threads: usize,
```

Comment:
I am thinking we should not make every config in this Struct, maybe abstract some scopes

Reply:
The current 'Config' is so flat ? Like put the same usages members into new struct:

pub struct xds_config {
 /// Filepath to a local xds file for workloads, as YAML.
    pub local_xds_path: Option<String>,
    /// If true, on-demand XDS will be used
    pub xds_on_demand: bool,"
}